components/edit-collective/EditMembers.js

```diff
@@ -158,13 +156,12 @@ class EditMembers extends React.Component {
   removeMember = index => {
     return this.setState(state => {
       const memberEntry = state.members[index];
-      if (memberEntry.member && !this.confirmRemoveMember(memberEntry)) {
-        return state;
-      } else {
+      if (!memberEntry.member || this.confirmRemoveMember(memberEntry)) {
```

Comment:
Same as above, I don't see any reason to change the condition, so let's revert and return null.